### PR TITLE
[RHPAM-1361] Add SSO principal-attribute

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

Fixes https://issues.jboss.org/browse/RHPAM-1361

Depends on https://github.com/jboss-openshift/cct_module/pull/270

### CODE REVIEW

File: templates/rhpam70-authoring-ha.yaml [hunk 1]
```diff
@@ -365,6 +365,10 @@ parameters:
   name: SSO_DISABLE_SSL_CERTIFICATE_VALIDATION
   value: "false"
   required: false
+- displayName: RH-SSO Principal Attribute
+  description: RH-SSO Principal Attribute to use as username. Defaults to preferred_username
```
Comment (jakubschwan) [inline templates/rhpam70-authoring-ha.yaml]: This environment variable do not have set default value (value: "preferred_username"). From the jira description [CLOUD-2722](https://issues.jboss.org/browse/CLOUD-2722) I would expect, that for an empty value of the SSO_PRINCIPAL_ATTRIBUTE parameter is used as default value "sub". @ruromero please, can you add the default value for all templates?

Comment (ruromero) [reply templates/rhpam70-authoring-ha.yaml]: @jakubschwan once the PR for CLOUD-2722 is merged I’ll apply the changes

**Comment by ruromero** [comment]
@spolti can you remove the _on hold_ label? It should be OK to merge now.

**Comment by jakubschwan** [comment]
@ruromero please create also PR with changes for 7.1 branch

**Comment by ruromero** [comment]
@jakubschwan done here https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/134

**Pull request merged**